Dfe.Academies.External.Web/Pages/Base/BasePageEditModel.cs

```diff
@@ -104,6 +104,11 @@ private void CacheSelectedSchool(EstablishmentResponse? schoolDetails)
 
 	protected DateTime BuildDateTime(string day, string month, string year)
 	{
+		if (!string.IsNullOrWhiteSpace(month) && month.Length == 1) // just '9' being passed in for instance !
```

Comment:
I would probably change this to use something like this

`
string dateString = $"{day.PadLeft(2, '0')}/{month.PadLeft(2, '0')}/{year.PadLeft(4 ,'0')}";
return DateTime.ParseExact(dateString, "dd/MM/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None);
`

Reply:
great call @jarednimble , fixed up